### PR TITLE
LPS-151134

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related Issue: https://issues.liferay.com/browse/LPS-151134

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:stop

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:stop

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java [hunk 1]
```diff
@@ -120,37 +128,57 @@ public Map<String, Object> getProperties(
...
+			objectFieldRenderingContext.getUserId());
+
+		if (folderId != null) {
+			try {
+				ModelResourcePermissionUtil.check(
+					_folderModelResourcePermission,
+					PermissionCheckerFactoryUtil.create(
+						objectFieldRenderingContext.getUser()),
+					objectFieldRenderingContext.getGroupId(), folderId,
+					ActionKeys.ADD_DOCUMENT);
```
[the remark this answers is not in the document]
Comment (rodrigopaulino) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java:159]: This is needed so we can pass the warning message to the component at first render. Perhaps we don't need the other one, but there's more to this whole permissions thing that I've been discussing with Product. Let's leave this discussion conversation open and I'll come back with more details later. Is that ok?
Thanks, @carolmariaabb

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java [hunk 2]
```diff
@@ -120,37 +128,57 @@ public Map<String, Object> getProperties(
...
+						objectFieldRenderingContext.getLocale(),
+						"you-do-not-have-permission-to-access-this-field-on-" +
+							"this-object"));
+			}
+		}
+
+		properties.put(
+			"objectEntryId", objectFieldRenderingContext.getObjectEntryId());
+		properties.put("objectFieldId", objectField.getObjectFieldId());
+		properties.put("portletId", objectFieldRenderingContext.getPortletId());
```
[the remark this answers is not in the document]
Comment (rodrigopaulino) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java:180]: Consider it done.

**Comment by rodrigopaulino** [comment]
ci:close

**Pull request closed without merging**